fastlane/fastlane/Fastfile

```diff
@@ -185,8 +185,8 @@ private_lane :print_release_changelog do |options|
   tool_name = options[:tool]
   old_version = options[:old_version]
 
-  cmd = "git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges .."
-  puts "Changes since release #{old_version}:\n" + `#{cmd}`.gsub("\n\n", "\n")
+  changes = sh("git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges ..").gsub("\n\n", "\n")
```

Comment:
I believe this change behaves differently than the original, and will not work. 😞 

```ruby
task :blah do
  value = sh('pwd')
  puts "Value: #{value}"
end
```

```
$ rake blah
pwd
/Users/mfurtak/Documents/Programming/Ruby/fastlane
Value: true
```

`sh()` always prints the the command and the output, but returns `true`/`false` 😫